### DISCUSSION TITLE
Sleuth Kit installation issues - 404 Not found

### DISCUSSION THREAD

**Discussion by Citizen839X** [community/community#103231] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi everyone,

I'm trying to install Autopsy but when I go down the process to install the missing Sleuth Kit part I got this issue

~/Downloads/sleuthkit-sleuthkit-4.12.1> sh travis_install_libs.sh
+ installLib libvhdi 20201204
+ wget https://github.com/libyal/libvhdi/releases/download/20201204/libvhdi-alpha-20201204.tar.gz
--2024-02-03 12:42:25--  https://github.com/libyal/libvhdi/releases/download/20201204/libvhdi-alpha-20201204.tar.gz
Resolving github.com (github.com)... 140.82.121.4
Connecting to github.com (github.com)|140.82.121.4|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-02-03 12:42:25 ERROR 404: Not Found.

I'm not a coder, any help would be immensely appreciated.

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
